hw1_4.py

```diff
@@ -0,0 +1,15 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов.
+# Вместе они сделали S журавликов.
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа
+# сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов,
+# чем Петя и Сережа вместе?
+
+
+s = int(input('Сколько журавликов сделали дети? '))
+# Пусть Петя и Сережа сделали по a журавликов, тогда Катя сделала 2*2a=4a журавликов,
+# следовательно общее количество журавликов должно быть кратно 6 (a+a+4a=6a)
+if s % 6 == 0:
+    a = int(s / 6)
```

Comment:
Строго говоря, если `s` точно кратно 6, а у вас так и будет, так как иное число в этот блок кода не попадёт, функцию int использовать не обязательно. Однако, если вам мешает ноль с точкой в выводе, можно сделать так:
```suggestion
    a = s // 6
```

Reply:
без этого ответ был float почему-то

Reply:
Если использовать оператор целочисленного отделения, то в результате будет тип данных `int`.